tests/acceptance/stepDefinitions/webdavContext.js

```diff
@@ -72,7 +72,8 @@ Then('as {string} file/folder {string} should not exist on remote server', funct
   return backendHelper.runOnRemoteBackend(fileShouldNotExist, userId, element)
 })
 
-Then('as {string} file/folder {string} should exist', function(userId, element) {
+Then('as {string} file/folder {string} should exist', async function(userId, element) {
+  await client.pause(1000)
```

Comment:
methods like `waitForAjaxCallsToStartAndFinish()` or `waitForFileVisible(filename)` could be more efficient than static wait, if we can use it here.

Reply:
Since the function makes api call I don't think we can use those functions.

Reply:
can we do `retry`? 🤔 the static wait is not good for automated tests unless exceptions (this may be)